Assets/Scripts/Camera/CameraMovement.cs

```diff
@@ -1,22 +1,21 @@
-using System.Collections;
-using System.Collections.Generic;
 using UnityEngine;
 
 public class CameraMovement : MonoBehaviour
 {
-    float horizontalSpeed = 2.0f;
-    float verticalSpeed = 2.0f;
-    public float verticalRotation = 0f;
-    public float horizontalRotation = 0f;
+    public float horizontalSpeed = 2.0f;
+    public float verticalSpeed = 2.0f;
```

Comment:
They determine the mouse sensitivity, so we were reluctant to make them final or private, in case we decide to add a feature to adjust mouse sens in the future

Reply:
As I said: Then document it.

Reply:
changed